### ISSUE TITLE
final part of installation

### ISSUE THREAD

**Issue by WillowCharcoal** [simnibs/simnibs#353]
Hi, when go to install the simnibs using the windows installer on win 11 it installs the environment and documentation but not the actual application and this is the installation text document
[install.log](https://github.com/user-attachments/files/15516699/install.log)

the error is:
     class TmsCoilDeformation(ABC, TcdElement):
  File "C:\Users\tlhc0\SimNIBS-4.1\simnibs_env\lib\site-packages\simnibs\simulation\tms_coil\tms_coil_deformation.py", line 95, in TmsCoilDeformation
    def apply(self, points: npt.NDArray[np.float_]) -> npt.NDArray[np.float_]:
AttributeError: module 'numpy.typing' has no attribute 'NDArray'

If anyone could help me resolve this that would be great.

**Comment by oulap** [comment]
Hey,

I tried this on a win 11 computer and the installation was successful. Could you try to do a clean install from start? Also if you have other simnibs versions lying around please remove those.

**Comment by WillowCharcoal** [comment]
This hasn't worked either, thanks for the suggestion.

**Comment by oulap** [comment]
It looks like the error is in the postinstallation script, which means that you probably have a somewhat working simnibs installation. You said that the environment installs correctly, could you launch the python interpreter, which is inside the environment, and then try to import simnibs?

**Comment by WillowCharcoal** [comment]
Unfortunately that still doesn't seem to work it comes up with the same error.

**Comment by oulap** [comment]
But you were able to start the simnibs python interpreter? If yes, then please try to upgrade the numpy version using pip and see if that solves the problem.

**Comment by WillowCharcoal** [comment]
Yeah I could open the interpreter, but upgrading numpy doesn't seem to change anything.

**Comment by oulap** [comment]
Can you dump the package versions you have installed in the simnibs environment? I.e., what's the printout from

conda env list
